recipes/fruit/all/conanfile.py

```diff
@@ -21,26 +21,37 @@ class FruitConan(ConanFile):
     topics = ("injection", "framework")
     settings = "os", "compiler", "build_type", "arch"
     options = {"shared": [True, False],
-               "use_boost": [True, False],
+               "use_boost": [True, False, "deprecated"],
+               "with_boost": [True, False],
                "fPIC": [True, False]}
-    default_options = {"shared": False, "use_boost": True, "fPIC": True}
+    default_options = {
+        "shared": False,
+        "use_boost": "deprecated",
+        "with_boost": True,
+        "fPIC": True}
 
     def export_sources(self):
         export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def package_id(self):
+        del self.info.options.use_boost
 
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+        if self.options.use_boost != "deprecated":
+            self.output.warn("use_boost option is deprecated, do not use anymore.")
```

Comment:
```suggestion
            self.output.warn("use_boost option is deprecated, use the option with_boost instead.")
```
You should point out which is the option to use instead.